src/include/citus_config.h.modified

```diff
@@ -0,0 +1,99 @@
+/* src/include/citus_config.h.  Generated from citus_config.h.in by configure.  */
```

Comment:
I believe those files are generated by `make reindent`, makes sense to exclude those `.modified` files